komodo-relational/src/main/java/org/komodo/relational/teiid/CachedTeiid.java

```diff
@@ -72,10 +89,10 @@
          */
         @Override
         public CachedTeiid create( final UnitOfWork transaction,
-                             final Repository repository,
-                             final KomodoObject parent,
-                             final String id,
-                             final RelationalProperties properties ) throws KException {
+                                   final Repository repository,
+                                   final KomodoObject parent,
+                                   final String id,
+                                   final RelationalProperties properties ) throws KException {
             return new CachedTeiidImpl(transaction, repository, id);
```

Comment:
Its even more crucial in a 30 file commit since the wading through of reformatting changes obscures the 'real' changes and makes the PR far harder to review.